usr/share/rear/finalize/Linux-ppc64/540_check_lilo_path.sh

```diff
@@ -0,0 +1,28 @@
+# THIS SCRIPT CONTAINS PPC64 SPECIFIC
+#################################################################
+# The purpose of this script is to check if the "part" variable in /etc/lilo.conf
+# is an existing disk partition on the local system. If not replace it with the PReP
+# partition path.
+#
+# This script must be run before 610_intall_lilo.sh and 550_rebuild_initramfs.sh
+#################################################################
+
+# skip if lilo conf is not found
+test -f $TARGET_FS_ROOT/etc/lilo.conf || return
+
+# Find PPC PReP Boot partitions
+part=$( awk -F '=' '/^boot/ {print $2}' $TARGET_FS_ROOT/etc/lilo.conf )
```

Comment:
I'm not a big fan of `sed` and usually prefer `awk` especially for this kind of case where in need to parse and get a data located in a specific field/column.
I could also use `cut` for that (may be easier for everyone to understand).
But I don't know how to do it with `sed`. Just tell me and I will do it.